### DISCUSSION TITLE
洼地底层人，目前民办三本，没钱没技能，怎么润?

### DISCUSSION THREAD

**Discussion by LeonorNeo** [The-Run-Philosophy-Organization/run#325] [General]
我今年大三，高考为了上个大学学了个美术，考了个民办三本。但对美术和设计并不感兴趣，水平很差，可能学校教学不行。唯一好点的是英语水平还可以，六级500+，下一步打算，在润前考雅思或托福。但没有启动资金润。好在家里认识关系可以让我去上海厂里当牛马，三班倒，但有晋升渠道。毕业去干月薪最高1w，包吃住。目前打算在国内工作三年攒启动资金，但出国之后的打算没有想好。初步考虑是技术移民，但自己又什么都不会，希望各位过来人给提提建议，拓展一下我的视野。我真的好想润，脱离苦海，开启真正自由的人生。

**Comment by LeonorNeo** [comment]
求求大佬们给支支招吧，我在国内快疯了😭

**Comment by the-run-philosophy** [comment]
只会技术的话可以看看本repo下关于澳大利亚移民的部分

**Reply by LeonorNeo** [reply]
好的，谢谢老哥

**Comment by the-run-philosophy** [comment]
澳大利亚，日本等地都是pro 技术移民的。门槛比较低
